internal/nostr/nostr.go

```diff
@@ -794,6 +794,10 @@ func (svc *Service) handleResponseEvent(event *nostr.Event, subscription *Subscr
 		"request_event_id": subscription.RequestEvent.ID,
 		"wallet_pubkey":    svc.getWalletPubkey(subscription.Authors),
 	}).Info("Received response event")
+	if (subscription.RequestEvent != nil) {
```

Comment:
why is it here `RequestEvent` and then `RequestEventDB`?

Reply:
Opened #98 to tackle this separately

Reply:
Done